File: check_log.py
import paramiko

client = paramiko.SSHClient()
client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
try:
    client.connect('192.168.10.201', 22, username='fo3nix', password='Testonly.3a', timeout=10)
    
    # Read log
    stdin, stdout, stderr = client.exec_command('cmd /c type "C:\\Users\\fo3nix\\relay_sidecar.log"')
    out = stdout.read().decode('gbk', errors='replace')
    print('Log:', repr(out))
    
    client.close()
except Exception as e:
    print('Error:', e)

File: check_log.js
const {Client} = require('ssh2');

const conn = new Client();
conn.on('ready', () => {
  const cmd = 'powershell -Command "Get-Content C:\\Users\\fo3nix\\relay_sidecar.log -Tail 10 -ErrorAction SilentlyContinue"';
  conn.exec(cmd, (err, stream) => {
    if (err) { console.error('exec err:', err); conn.end(); return; }
    let out = '';
    stream.on('data', d => out += d);
    stream.on('close', () => { console.log(out||'empty'); conn.end(); });
  });
}).on('error', e => console.error('conn err:', e.message)).connect({
  host:'192.168.10.201', port:22, username:'fo3nix', password:'Testonly.3a'
});
